src/Patch/Element.js

```diff
@@ -116,6 +125,44 @@ export default function(internals) {
     });
   }
 
+  function patch_outerHTML(destination, baseDescriptor) {
+    patch_HTMLsetter('outerHTML', destination, baseDescriptor, (node, parentNode, patchFunction) => {
+      patchFunction.call(internals, parentNode);
+    });
+  }
+
+  if (Native.Element_outerHTML && Native.Element_outerHTML.get) {
+    patch_outerHTML(Element.prototype, Native.Element_outerHTML);
+  } else if (Native.HTMLElement_outerHTML && Native.HTMLElement_outerHTML.get) {
+    patch_outerHTML(HTMLElement.prototype, Native.HTMLElement_outerHTML);
+  } else {
+
+    internals.addPatch(function(element) {
+      patch_outerHTML(element, {
+        enumerable: true,
+        configurable: true,
+        // Implements getting `outerHTML` by performing an unpatched `cloneNode`
+        // of the element and returning the resulting element's `outerHTML`.
+        // TODO: Is this too expensive?
+        get: /** @this {Element} */ function() {
+          return Native.Node_cloneNode.call(this, true).outerHTML;
+        },
+        set: /** @this {Element} */ function(assignedValue) {
+          const container = Native.Document_createElement.call(document, 'div');
```

Comment:
Or maybe `createElementNS(this.parentNode.namespaceURI, ...`?

Reply:
In this case it is not necessary, as `div` is always in the HTML namespace :smile:

Reply:
True :) , more of what I'm getting at though is that this seems like this would need to do something like [this segment from the `innerHTML` wrapper](https://github.com/webcomponents/custom-elements/blob/a198647cc0877a30c4ab930cf8b0d3ba8e485ad7/src/Patch/Element.js#L103-L105) so that `outerHTML` creates elements in the namespace of the parent.